amr-wind/utilities/io.cpp

```diff
@@ -96,6 +103,13 @@ void incflo::ReadCheckpointFile()
         }
     }
 
+    if (replicate) {
+        for (int d = 0; d < BL_SPACEDIM; d++) {
+            prob_lo[d] = Nrep[d] * prob_lo[d];
```

Comment:
This seems like it can lead to confusion. Since restart files have the old prob lo/hi would it make more sense to compare user provided lo/hi and compute nrep?

Reply:
Also this only works as advertised if problo is (0,0,0). Otherwise it seems that your domain will be 2x times larger and cells coarser

Reply:
yeah it is confusing, I'll change it up. Thanks!